app/controllers/admin_request_controller.rb

```diff
@@ -193,6 +193,7 @@ def info_request_params
     if params[:info_request]
       params.require(:info_request).permit(:title,
                                            :prominence,
+                                           :prominence_reason,
```

Comment:
Need to also record this in the hide/edit event on change as per https://github.com/mysociety/alaveteli/issues/6746#issuecomment-1133038351:

> When `InfoRequest#prominence_reason` is updated, it should log the prominence reason on the associated `edit` `InfoRequestEvent`.

![Screenshot 2022-09-30 at 12 04 25](https://user-images.githubusercontent.com/282788/193256765-d30b4174-7f80-4682-80fc-6bad3620938f.png)

Note that we'll also want to record this on `hide` events too.

https://github.com/mysociety/alaveteli/blob/0.41.1.0/app/controllers/admin_request_controller.rb#L48

https://github.com/mysociety/alaveteli/blob/0.41.1.0/app/controllers/admin_request_controller.rb#L160

Reply:
Both hide and edit events record the prominence and reason changes now.